irods/manager/data_object_manager.py

```diff
@@ -110,7 +109,8 @@ def _download(self, obj, local_path, num_threads, **options):
 
 
     def get(self, path, local_path = None, num_threads = DEFAULT_NUMBER_OF_THREADS, **options):
-        """Get a reference to the data object at the specified `path'.
+        """
+        Get a reference to the data object at the specified `path'.
```

Comment:
is the character after `path` supposed to be a backtick or single quote?

Reply:
It's a stylized thing sometimes used to set off a word visibly.  I should probably leave out both "quote" characters, esp if it tends to cause distraction or double-take... @trel?

Reply:
you're saying mixing them is on purpose?

what i see is `<backtick>path<single_quote>`.

Reply:
Yes, it was on purpose.  It seemed appropriate but may not be :)

Reply:
Understood. I think it matters for the next developer more than someone like me. If other developers understand it then it is likely ok.

Is this specific to python?

Reply:
seems fine if it renders cleanly wherever the docstrings get rendered...

Reply:
Not particularly Python-related, no. I've seen it in various spots around the internet, sometimes in documentation, probably more to do with style than accuracy.

Reply:
@trel yes it seems to render fine ....